ghci/containers/ghci-koji/ci/run-koji.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -eux
+
+# Set DB credentials
+sed -i  -e "s/.*DBHost =.*/DBHost = ${POSTGRES_HOST}/" \
+        -e "s/.*DBUser =.*/DBUser = ${POSTGRES_USER}/" \
+        -e "s/.*DBPass =.*/DBPass = ${POSTGRES_PASSWORD}/" \
+        -e "s/.*DBName =.*/DBName = ${POSTGRES_DB}/" \
+        /etc/koji-hub/hub.conf
```

Comment:
Where do these env-variables come from?

Reply:
They need to to be passed like this:

```
docker run -e POSTGRES_USER=koji koji
````

I don't want to hardcode any credentials into the image because I think that specifying the credentials should be a job for a container orchestrator which is also responsible for spinning up a postgres container.